packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/evaluate-expression.ts

```diff
@@ -19,7 +20,7 @@ export interface EvaluateExpressionProps extends sfn.TaskStateBaseProps {
   /**
    * The runtime language to use to evaluate the expression.
    *
-   * @default lambda.Runtime.NODEJS_14_X
+   * @default lambda.Runtime.NODEJS_16_X
```

Comment:
Better not promise this, eh :)

Reply:
yeah made this more explicit that it defaults to the region's default? Kinda vague still. I guess the default default is node16?